### PR TITLE
Daily reading

### LINKED ISSUE #16: Random Card Display
- [ ] Get website to talk to website. 
- [ ] Get a random card's info to display onto the webpage.

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description
This pr adds styling adjustments, columns and rows to the website. The site will now display a loader until the fetch request is complete and the state is filled with the card data from the empress-tarot api. App now has a function to grab a random card, and that card is passed into the 'daily reading' component to display its info. If the card is reversed, the image will display upside down.

More testing is required.

Prop Types need to be added

## Screenshots

Screen Shot 2020-10-28 at 6 02 41 PM

Screen Shot 2020-10-28 at 6 02 28 PM

## How Has This Been Tested?

- npm start in Chrome, and with Chrome developer tools. More testing is needed

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Issues

- Closes #16

### CODE REVIEW

File: src/DailyReading/DailyReading.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+h1, h2, h3, p {
+  color: white
+}
+
+.rotateimg180 {
+  -webkit-transform:rotate(180deg);
```
Comment (chadrick-d-dev) [inline src/DailyReading/DailyReading.css:6]: Which one do we need?

File: src/DailyReading/DailyReading.js [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
-  return (
-    <div>
-      <Button variant="success">Test Button</Button>
-    </div>
-  )
+export default function DailyReading(props) {
+  if (!props.card.reversed) {
+    return (
+      <div>
+        <Row>
```
Comment (chadrick-d-dev) [inline src/DailyReading/DailyReading.js:9]: New tag for me! Never used Row or Col tags

**Review by chadrick-d-dev — APPROVED** [review]
We got so much done guys! Great job team!

**Review by kathrynljackson — COMMENTED** [review]
Is the alert imported from React different from the one that your can just manually add? I've never seen it imported before!

**Pull request merged**